SignalsIntegrationTests/SignalsIntegrationTests/BasicSignalTests.cs

```diff
@@ -30,6 +30,16 @@ public void RequestForNonExistingSignalThrowsOrReturnsNull()
         }
 
         [TestMethod]
+        public void RequestForSettingFirstOrderMissingValuePolicyForStringThrows()
+        {
+            var signalId = AddNewStringSignal().Id.Value;
+
+            Assertions.AssertThrows(() => client.SetMissingValuePolicy(
```

Comment:
`[ExpectException(...)]`? Why introducing own `AssertThrows`? (it might be more readable but some tests already use `Expect`)